app/sdil/controllers/ContactDetailsController.scala

```diff
@@ -62,39 +62,41 @@ class ContactDetailsController(val messagesApi: MessagesApi, cache: Registration
   }
 }
 
-object ContactDetailsController extends FormHelpers {
+object ContactDetailsForm extends FormHelpers {
```

Comment:
If we're changing this to be called a form it's probably better to make the apply method work as the form rather than declaring a separate val inside the object